src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain
+
+import kotlin.random.Random
+
+class RacingGame(
+    val carSize: Int,
+    val attemptSize: Int
+) {
+    private val cars = List(carSize) { Car() }
+    var curAttemptNum: Int = 0
+        private set
+    private val _history: MutableList<List<Int>> = mutableListOf()
+    val history: List<List<Int>>
+        get() = _history
```

Comment:
```suggestion
    private val _history: MutableList<List<Car>> = mutableListOf()
    val history: List<List<Car>>
        get() = _history
```

history에 대한 부분을 Int가 아닌 자동차 객체를 반환해주는 건 어떨까요?